### DISCUSSION TITLE
Launch of the "Registries of Good Practice" project

### DISCUSSION THREAD

**Discussion by anjackson** [digipres/digipres.github.io#45] [Announcements]
I'm now working for the Digital Preservation Coalition, on the Registries of Good Practice Project. There's an announcement [here](https://www.dpconline.org/news/registries-of-good-practice).

Part of the plan is to more actively maintain and update the shared resources managed through this GitHub organisation.  So far, the proposal is:

- That I will more actively maintain and update these resources! Keep an eye on this forum, process any PRs, update the Awesome List a bit. That kind of thing.
- That the project itself will run our of this organisation, transferring over the existing repositories currently running out of my persional account: https://github.com/anjackson/digipres-practice-project and https://github.com/anjackson/digipres-practice-index
- That the outputs of this project will run out of the digipres.org GitHub pages (where appropriate), like the other awesome-list/formats/policies things do.

If you have any questions or concerns about this, please comment below or [email me](mailto:andrew.jackson@dpconline.org).

**Comment by anjackson** [comment]
Some more information on the inital goals and principles of the project [here](https://www.dpconline.org/blog/registries-of-practice-goals-principles).

**Comment by ross-spencer** [comment]
It seems to me like a net positive to build up the github.com/digipres org @anjackson :+1:
